src/components/routes/Account/AccountNav/AccountNav.container.ts

```diff
@@ -12,6 +12,7 @@ const AccountNavContainerMobile = styled.nav`
     background-color: ${color('light')};
     color: ${color('secondary')};
     display: flex;
+    flex-basis: 0;
```

Comment:
is this the CSS just for the tab component?

we should avoid specific names like `verification-needed` so we can just re-use it as different use cases instead of copying and pasting.
it can be a generic name like`alert-badge`

cc @kcvan

Reply:
changed `verification-badge` to `alert-badge`
changed `<AccountNav />` to `<ReusableNavBar />`